recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+
+git clone https://github.com/twosigma/beakerx.git
+cd beakerx/
+git checkout 585f07c5dfe7f9f0f97053d90ffc5a696d972382
```

Comment:
Beakerx is a main dependency of this package. That being said all dependencies are managed by Maven. The thing is for now the Beakerx package is missing a Maven artifact I could use.

So I would like to merge it like this to start with. So people can start using the kernel and give me feedback on it.

Then once the Beakerx project release Maven artifacts (probably in few weeks), I will modify this recipe to remove the manual Beakerx installation.

Hope you understand.